src/main/cpp/include/vcf/genomicsdb_bcf_generator.h

```diff
@@ -82,7 +82,7 @@ class GenomicsDBBCFGenerator
     VariantQueryConfig m_query_config;
     VCFSerializedBufferAdapter m_vcf_adapter;
     unsigned m_query_column_interval_idx;
-    VariantQueryProcessorScanState m_scan_state;
+    VariantQueryProcessorScanState* m_scan_state;
```

Comment:
Please revert this back  - the bug is in a different code segment and this will not fix it.